locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -651,7 +651,7 @@ protected MachineLocation obtainOnce(ConfigBag setup) throws NoMachinesAvailable
         Stopwatch provisioningStopwatch = Stopwatch.createStarted();
 
         try {
-            LOG.info("Creating VM "+setup.getDescription()+" in "+this);
+            LOG.info("Creating VM "+getCreationString(setup)+" in "+this);
```

Comment:
[minor / ignore] I would change this to  `LOG.info("Creating VM {} in {}", getCreationString(setup), this);`

probably applies elsewhere